source/Playnite/Database/GameDatabase.cs

```diff
@@ -753,7 +753,13 @@ public List<Game> ImportGames(LibraryPlugin library, bool forcePlayTimeSync)
                 }
             }
 
-            return addedGames;        
+            // Set the uninstalled property for games that were removed
+            foreach (var game in Games.Where(g => g.PluginId == library.Id && g.IsInstalled))
+            {
+                game.IsInstalled = libraryGames.Any(lg => lg.GameId == game.GameId && lg.IsInstalled);

```

Comment:
You need to call `Games.Update` otherwise the information won't be persistent between app launches. Also only call it when the actual `IsInstalled` state changes (ideally don't even do the assignment to prevent property change notification to UI).

Reply:
Yes, that was a mistake on my part, sorry.

I have pushed a fix that changes the code to behave as you have described.